src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -13,11 +13,13 @@
 import requests
 
 from knack.util import CLIError
-from azure.cli.core.prompting import prompt, prompt_pass, NoTTYException
+from knack.prompting import prompt, prompt_pass, NoTTYException
+from knack.log import get_logger
 
+from ._client_factory import cf_acr_registries
 from ._constants import MANAGED_REGISTRY_SKU
 from ._utils import get_registry_by_name
-from .credential import acr_credential_show
+from .credential import _get_acr_credentials
```

Comment:
NIT: should non-private method starts with underscore `_`?

Reply:
Changed to `get_acr_credentials`